### PR TITLE
Removing unused variables

### PR DESCRIPTION
**Pull request by ericcurtin**

### CODE REVIEW

File: src/libostree/ostree-sysroot-deploy.c [hunk 1]
```diff
@@ -2135,8 +2131,7 @@ ostree_sysroot_deploy_tree (OstreeSysroot     *self,
         goto out;
     }
 
-  { ostree_cleanup_sepolicy_fscreatecon gpointer dummy = NULL;
-
+  {
```
Comment (cgwalters) [inline src/libostree/ostree-sysroot-deploy.c]: This *is* used to unset the context state.   We likely need to annotate `G_GNUC_USED` or so.

**Comment by cgwalters** [comment]
I'm guessing you're building with CLang?  This is likely going to conflict with some outstanding patches, but we can deal.

Can you [improve the commit message](http://chris.beams.io/posts/git-commit/) ?  See the existing bits in `git log`.  Something like this as a start:
```
build: Remove unused variables

I noticed this when building with CLang.
```

**Comment by ericcurtin** [comment]
Done :)

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ 8cddf98

**Comment by cgwalters** [comment]
@ericcurtin You're going to need to do a build with all options enabled for this (selinux, libarchive, etc.).

**Comment by ericcurtin** [comment]
Ah yes I see there are things #ifdef'd out... I'll get to it this evening

**Comment by cgwalters** [comment]
Fixed by https://github.com/ostreedev/ostree/pull/548

**Pull request closed without merging**